Studio-Lovelies/GG-JointJustice-Unity
# Feature/70 story handler

## Summary

Implements story controller. Identical to [this PR](https://github.com/Studio-Lovelies/GG-JointJustice-Unity/pull/82) except with some added polish.

"Stories are now handled by the story handler, which can transition between different inky dialogue scripts. It also loads the next unity scene when reaching the end"

## Testing instructions

Run the ink test scene and make sure the narratives scripts transition into each other. Also run the tests.
Note: the screenshake doesn't stop when continuing to the next narrative script. This could be considered a bug (and it will need to be fixed in another PR if so), though I suspect screenshakes wont be that long usually, so it might not matter.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
closes #70
